doc/customization_validation.rst

```diff
@@ -0,0 +1,59 @@
+.. SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _customization_validation:
+
+###################
+4. Model Validation
+###################
+
+To validate the data obtained with PyPSA-Earth, we recommend to go through the procedure here detailed. An exampled of the validation procedure is available in the `Nigeria validation <https://github.com/pypsa-meets-earth/documentation/blob/main/notebooks/validation/validation_nigeria.ipynb>`_ notebook. Public information on the power system of Nigeria are compared to those obtained from the PyPSA-Earth model.
+
+Simulation procedure
+^^^^^^^^^^^^^^^^^^^^
+
+It may be recommended to check the following quantities in the validation:
+
+#. Inputs used by the model:
+
+    #. Network characteristics
+
+    #. Substations
+
+    #. Installed generation by type
+
+#. Outputs of the simulation:
+
+    #. Demand
+

```

Comment:
Demand is in fact an output of a computational procedure. But that is also an input for an optimisation model, which can actually use also measured demand time series. Can we probably add a little clarification on that to avoid confusion?